src/main/java/org/jabref/logic/importer/fetcher/ACMPortalFetcher.java

```diff
@@ -38,23 +36,32 @@ public String getName() {
         return Optional.of(HelpFile.FETCHER_ACM);
     }
 
-    private static String createQueryString(QueryNode query) throws FetcherException {
+    private static String createQueryString(QueryNode query) {
         String queryString = new DefaultQueryTransformer().transformLuceneQuery(query).orElse("");
-        // Query syntax to search for an entry that matches "one" and "two" in any field is: (+one +two)
-        return "(%252B" + queryString.trim().replaceAll("\\s+", "%20%252B") + ")";
+        return queryString.trim().replaceAll("\\s+", "+");
     }
 
+    /**
+     * Constructing the url for the searchpage.
+     *
+     * @param query query node
+     * @return query URL
+     */
     @Override
-    public URL getURLForQuery(QueryNode luceneQuery) throws URISyntaxException, MalformedURLException, FetcherException {
+    public URL getURLForQuery(QueryNode query) throws URISyntaxException, MalformedURLException, FetcherException {
         URIBuilder uriBuilder = new URIBuilder(SEARCH_URL);
-        uriBuilder.addParameter("query", createQueryString(luceneQuery)); // Search all fields
-        uriBuilder.addParameter("within", "owners.owner=GUIDE"); // Search within the ACM Guide to Computing Literature (encompasses the ACM Full-Text Collection)
-        uriBuilder.addParameter("expformat", "bibtex"); // BibTeX format
+        uriBuilder.addParameter("AllField", createQueryString(query));
         return uriBuilder.build().toURL();
     }
 
+    /**
+     * Gets an instance of ACMPortalParser.
+     *
+     * @return ACMPortalParser
```

Comment:
In case you cannot state something about a return type - please just remove that JavaDoc line

Reply:
fixed.